src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -47,13 +47,22 @@ angular
     '$$rAF',
     '$window',
     '$mdProgressCircular',
+    '$mdTheming',
```

Comment:
Just a question related to the progress circular. Why are we using Strict DI here?